### DISCUSSION TITLE
WTN-Project

### DISCUSSION THREAD

**Discussion by MoYonsei** [spatie/laravel-permission#2652] [General]
I have created a software project, which is a Laravel website with the laravel breeze system, and I have recently created a system, and there is also this spatie/laravel-permission package for permissions, and I added moderators via databaseseeder.php, they are admin and super admin so that when they log in, they go to this admin link/ and created in the account creation form an entry for the user type if freelnacer goes to the packages page if they are available and if they are not available, it goes to the dashboard/ and if the client goes to home, but there is an error because when super admin and admin log in, they go to the dashboard/ or home/instead of admin/ either the problem the second is in a special payment system so that The site has a package system, paypal and Stripe payment system, and the administrator enters the account keys via the control panel and they are saved in the database

https://github.com/MoYonsei/WTN-Project

**Comment by drbyte** [comment]
`super_admin` role is created twice:
https://github.com/MoYonsei/WTN-Project/blob/8d8cb15e00bcde2c3cf76ee6bfbc2b20f9c319cf/database/seeders/DatabaseSeeder.php#L46-L50

https://github.com/MoYonsei/WTN-Project/blob/8d8cb15e00bcde2c3cf76ee6bfbc2b20f9c319cf/database/seeders/RolePermissionSeeder.php#L25-L26

role to `access dashboard` is duplicated with `accessdashboard`:
https://github.com/MoYonsei/WTN-Project/blob/8d8cb15e00bcde2c3cf76ee6bfbc2b20f9c319cf/database/seeders/RolePermissionSeeder.php#L21

https://github.com/MoYonsei/WTN-Project/blob/8d8cb15e00bcde2c3cf76ee6bfbc2b20f9c319cf/database/seeders/RolePermissionSeeder.php#L41

`admin dashboard route` is defined twice:
https://github.com/MoYonsei/WTN-Project/blob/8d8cb15e00bcde2c3cf76ee6bfbc2b20f9c319cf/routes/web.php#L50-L51

https://github.com/MoYonsei/WTN-Project/blob/8d8cb15e00bcde2c3cf76ee6bfbc2b20f9c319cf/routes/web.php#L129-L131

`redirectTo` is defined differently in two places:
https://github.com/MoYonsei/WTN-Project/blob/8d8cb15e00bcde2c3cf76ee6bfbc2b20f9c319cf/app/Providers/RouteServiceProvider.php#L28-L43

https://github.com/MoYonsei/WTN-Project/blob/8d8cb15e00bcde2c3cf76ee6bfbc2b20f9c319cf/app/Http/Controllers/Auth/LoginController.php#L22-L35

I wonder if these duplications and differences are causing the unexpected results?
